setup.py

```diff
@@ -119,6 +119,7 @@ def find_version(*file_paths):
     # Using 2.x versions of pyparsing seems to fix the issue.
     # Ticket: 69520
     "pyparsing<3.0",
+    "rich>=13.5.2",
```

Comment:
Should we remove tqdm from setup.py and replace all tqdm?

Reply:
I think it's better to try out `rich` for some time. Then, if everything works out fine we will replace all `tqdm` calls.